protocol/migrations/sqlite/1683725608_add_token_balances.up.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE IF NOT EXISTS token_balances (
+  user_address VARCHAR NOT NULL,
+  token_name VARCHAR NOT NULL,
+  token_symbol VARCHAR NOT NULL,
+  token_address VARCHAR NOT NULL,
+  token_color VARCHAR NOT NULL DEFAULT "",
+  token_decimals INT NOT NULL,
+  token_description VARCHAR NOT NULL DEFAULT "",
+  token_url VARCHAR NOT NULL DEFAULT "",
+  balance INT NOT NULL,
```

Comment:
Is INT enought?

Reply:
SQLite as far as I understand does not make difference between INT/BIGINT etc, they are all stored in the same way.
That would give us 2^63-1 as the maximum value ( 9,223,372,036,854,775,807 ), do you think that's enough?
Otherwise it would be a varchar, what do you think?

Reply:
probably best to store it as a varchar then